docs/index.org

```diff
@@ -480,11 +480,15 @@ subject of our metadata. Our metadata will be keyed by the subject, so
 when querying a metadata service for the metadata of our asset, we
 will use the subject in our query.
 
-For phase-1 monetary scripts the standard for wallet metadata is that
+For tokens minted using phase-1 monetary scripts the standard for metadata is that
 the subject is equal to the hash of the policy concatenated to the
 base16-encoded asset name (which can be empty).
 
-For phase-2 Plutus Smart-Contracts the subject is equal to the
+For tokens minted using phase-2 monetary scripts (Plutus) the standard for metadata is that
```

Comment:
Ah great, this distinction between phase-1 and phase-2 has helped me out. I've been trying to find the correct words to distinguish between these.